bugbug/repository.py

```diff
@@ -139,6 +139,8 @@ def download_commits(repo_dir, date_from):
     global author_experience_90_days
     for commit in commits:
         author_experience[commit] = total_commits_by_author[commit.author]
+        if commit.ever_backedout:
+            continue
```

Comment:
See https://github.com/mozilla/bugbug/pull/188#discussion_r258457469, do as you were doing before, but adding a comment like I said.

And also, you'll need to support ignoring backed out commits for the 90 days experience too.